crates/live_kit_client/Cargo.toml

```diff
@@ -19,7 +19,7 @@ test-support = [
     "collections/test-support",
```

Comment:
Live-kit was already compilable on Linux after #6859, and it is being worked on in #7302. Are these changes needed?

Reply:
Unsure, but let me know when would be the best time to pull latest from `main` into the PR branch, so that live kit doesn't need patches